### DISCUSSION TITLE
Unable to Log Into My Account

### DISCUSSION THREAD

**Discussion by limitpointinf0** [community/community#147644] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Whenever I log into my account, I have an SMS sent to my phone for the OTP. Today, I ran into this error message while trying to authenticate using SMS. Now I'm unable to get the OTP and log into my account.

"We tried sending an SMS to your configured number, but we are not authorized to send SMS messages to this recipient. Please contact support if you continue to have problems."

How can I get this resolved?

**Comment by davevad93** [comment]
Hi @limitpointinf0 ,

As this is an account-related issue, it's recommended that you contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.
